datadog/resource_datadog_dashboard.go

```diff
@@ -5181,6 +5181,26 @@ func getApmStatsQuerySchema() *schema.Schema {
 	}
 }
 
+func buildDatadogApmStatsQueryColumn(terraformColumn map[string]interface{}) *datadogV1.ApmStatsQueryColumnType {
+
+	datadogColumn := datadogV1.NewApmStatsQueryColumnTypeWithDefaults()
+
+	if value, ok := terraformColumn["name"].(string); ok && len(value) != 0 {
+		datadogColumn.SetName(value)
+	}
+	if value, ok := terraformColumn["alias"].(string); ok && len(value) != 0 {
+		datadogColumn.SetAlias(value)
+	}
+	if value, ok := terraformColumn["cell_display_mode"].(datadogV1.TableWidgetCellDisplayMode); ok && len(value) != 0 {
```

Comment:
You may have to build these as well?

Reply:
@nmuesch `TableWidgetCellDisplayMode` and `WidgetSort` are enums, so I don't think there's anything to build --would it be better to set them as `.(string)` here?

Reply:
(Update: I tried `.(string)` and it throws a type error)

Reply:
You need to cast to string, and then call `datadogV1.TableWidgetCellDisplayMode` on the value.